CHANGES.md

```diff
@@ -1,3 +1,21 @@
+## edge-18.11.3
+
+* CLI
+  * **New** `linkerd routes` command displays per-route stats for services with
+    service profiles
+  * **Improved** Add `--open-api` to `linkerd profile` command, to read an
+    OpenAPI / Swagger specification file
+  * **Experimental** New `--ha` flag on `linkerd install`, for HA deployment of
+    the control plane (thanks @benjdlambert!)
+* Web UI
+  * **New** Top Routes page, served at `/routes`
```

Comment:
TIOLI might want to say "experimental" top routes page, as it's unpolished? Or we can just not mention it.